third_party/libpg_query/pg_functions.cpp

```diff
@@ -55,12 +55,17 @@ static void allocate_new(parser_state *state, size_t n) {
 
 void *palloc(size_t n) {
 	// we need to align our pointers for the sanitizer
-	auto aligned_n = ((n + 7) / 8) * 8;
+	auto allocate_n = n + sizeof(size_t);
+	auto aligned_n = ((allocate_n + 7) / 8) * 8;
```

Comment:
Don't we have a utility function for this somewhere? Align<T> or something?

Reply:
Yes, but since this is in the libpg_query library in the third_party folder we prefer not to link it back to the main DuckDB sources.

Reply:
That's what I figured, but thought I should ask.